install/0000_30_machine-api-operator_02_machine.crd.yaml

```diff
@@ -3,6 +3,8 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    exclude.release.openshift.io/internal-opernshift-hosted: "true"
```

Comment:
Is `opernshift` a typo?

Reply:
@mrunalp, it is :( 
Thank you for catching it. Updating.